people/benefits.md

```diff
@@ -44,56 +44,63 @@ The monthly budgets compound every month, so whatever you do not spend this mont
 
 You can request an increase in your allowance if you feel the current one doesn't cover all the expenses for you. You can spend a few months of future allowance balance. The maximum allowance balance you can keep is 18 months. That said, it's smart to keep at least a few months allowance, just in case something breaks.
 
-##### What about those Niteans that receive their salary in USD?
-
-In case you live in a country that has their economy tied to USD mora than EUR,
-it is recommended that you choose to receive your salary in USD in order to
-decrease the risk of currency fluctuations affecting your base salary.
-
-In this case, your allowance will also come in USD. The exact amount of USD
-is calculated on 1st of January for every year, based on the current exchange
-rate, and rounded up to the nearest ten. For 2018, the USD allowance budget is
-$150 / month.
+If they attended a conference in the previous year, each Nitean gets a €1000 additional budget on 1st of January every year.
 
+##### What about those Niteans that receive their salary in USD?
 
-## Software Licenses
+In case you live in a country that has their economy tied to USD mora than EUR, it is recommended that you choose to receive your salary in USD in order to decrease the risk of currency fluctuations affecting your base salary.
```

Comment:
`In case you live in a` -> I would just go with `If you live...` Also, redundancy with next paragraph.

typo `...tied to USD mora than EUR`

Reply:
Fixed `If you live` and the type. I don't understand what you mean with the redundancy?

Reply:
Ah, noted that the two paragraphs were starting with "in case/in the case" - so the redundancy. 

All good!